wiki/Registration/zh.md

```diff
@@ -0,0 +1,21 @@
+# 注册
+
+*警告: 任何时候拥有一个以上的osu！账户都违反 [osu! 规则](/wiki/Rules)!*
+
+1. 在 [安装osu!](/wiki/Installation) 并运行之后，osu!将提示你登入或之策。 假设你现在没有账户，请点击 `创建账户`。
+
+   ![游戏内注册界面](img/ingame-registration.jpg "The registration form")
+
+2. 填写表格中的字段。
+   - 用户名最多只能有15个字符，用户名只能由字母 (`a-z`, `A-Z`, `0-9`)，下划线 (`_`)，方括号(`[` 和 `]`)，破折号(`-`)，和空格(` `)组成。
+   - 对于电子邮件地址字段，请使用你的邮箱。这将在你需要重置密码时给你发送验证码。所以不要丢失你的邮箱。
+   - 对于密码字段，请使用不少于8个字符的密码。
+3. 一旦完成注册，点击 `1. Create my account`。你将会自动登录。
+
+## 接下来做什么
+
+请不要忘记认真阅读[规则](/wiki/Rules)
+
+一旦完成注册，你就可以开始节奏之旅了。你可以[添加谱面](/wiki/Installation#adding-beatmapsets)进行游玩，[制作皮肤](/wiki/Skinning)或者[制作谱面](/wiki/Beatmapping)。你也可以在[自我介绍子论坛](https://osu.ppy.sh/community/forums/8)来介绍自己。
```

Comment:
forum titles should be left in english, since they aren't translated on the site. so I think the link should be `Introductions 子论坛` ?

same thing with help forum in next paragraph

Reply:
what I mean is keep the title in English, but write "subforum" in chinese

for example French says "sous-forum Introductions"